pyInterface/package/utils/_waveDescThresUtils.py

```diff
@@ -49,3 +49,30 @@ def getWaveDescThresFromWaveList(waveListFileName, keyFiles):
 			lineNmb += 1
 		printInfo("read " + str(lineNmb) + " lines from wave list file " + "'" + waveListFileName + "'")
 	return waveDescThres
+
+
+def getWaveThresFromWaveList(waveListFileName, waveDescriptions):
+	printInfo("reading amplitude names and thresholds from wave list file "
+	          + "'" + waveListFileName + "'.")
+	waveDescThres = []
+	with open(waveListFileName, 'r') as waveListFile:
+		lineNmb = 0
+		for line in waveListFile:
+			if line[0] == '#':  # comments start with #
+				continue
+			line = line.replace('\n', '')
```

Comment:
Couldn't you use line.strip()? If you do this before the check for comments, lines starting with " #" would also be recognized as comment